### PR TITLE
Pluto min build followup

### PR DESCRIPTION
**Pull request by alexrichey**
Assorted cleanup for pluto. Reasons for each change are in the commit message bodies

### CODE REVIEW

File: products/pluto/README.md
[the remark this answers is not in the document]
Comment (alexrichey) [reply products/pluto/README.md:1]: @damonmcc @fvankrieken I guess this is a good issue to clarify - what are your thoughts about what belongs in a wiki, vs what's in a project-level README? 

Offhand, I would think that the README would contain more technical instructions, e.g. for the building the project. Though as we standardize to building via dcpy, those project level technical details should probably become fewer and fewer.

[the remark this answers is not in the document]
Comment (damonmcc) [reply products/pluto/README.md:1]: I like that idea of what a README should have

[the remark this answers is not in the document]
Comment (alexrichey) [reply products/pluto/README.md:1]: @damonmcc Cool - gonna take a pass at cleaning up the WIKI, then can clean up build instructions here

[the remark this answers is not in the document]
Comment (damonmcc) [reply products/pluto/README.md:1]: a valiant effort! wouldn't object to restoring the readme here and saving that for later

[the remark this answers is not in the document]
Comment (alexrichey) [reply products/pluto/README.md:1]: @damonmcc Update. I think the versioning was the only thing worth saving. I add bare-bones instructions for building. There's a lot to be desired. Incidentally, I think that a massive upside of centralizing `dcpy` will be reducing the amount of documentation to something manageable.

 But I think we also need to add some info about `pts`/`cama`. Or... @fvankrieken Should those be moved to recipes?

[the remark this answers is not in the document]
Comment (fvankrieken) [reply products/pluto/README.md:1]: I would say add them here for now, but most likely yes they belong more in data library. But data library sort of belongs in monorepo anyways...

**Review by damonmcc — APPROVED** [review]

**Pull request merged**